start/sw/web-app/django.md

```diff
@@ -10,130 +10,103 @@ Django is a high-level Python Web framework that encourages rapid development an
 Django is one of the most popular Python Web Frameworks. It has very useful web page. Visit it by clicking on [this link](https://www.djangoproject.com/). 
 Because you will need to work with Python packages, write this command to your command line.
 
-For Python:
-
-```
-sudo dnf install python-pip
-```
-
-or Python 3:
-
 ```
-sudo dnf install python3-pip
+$ sudo dnf install python3-pip
 ```
 
-# Get started
+## Get started
 
-## How to install virtualenv
+### How to install virtualenv
```

Comment:
Please, if possible, link to an our already existing content / section and do not duplicate content.

f.e.: Install [virtualenv](https://developer.fedoraproject.org/tech/languages/python/python-installation.html#using-virtualenv) - `/tech/languages/python/python-installation.html#using-virtualenv`